src/hooks/useBlockchain.js

```diff
@@ -17,7 +17,9 @@ import {
 import { getWalletWMOVRBalance, getWalletZoomBalance } from '../utils/wallet'
 import watchMarketEvents from 'utils/setupWatcher'
 
-const isLocal = process.env.NODE_ENV === 'development'
+
+//const isLocal = process.env.NODE_ENV === 'development'
+const isLocal = true;
```

Comment:
This seems dangerous. Is this temporary?

Reply:
yes. temporary.. is there a way to have just the server flip ? I created a .env and set the value.. but it doesnt work if I build for production. suggestions ?